control-plane/roles/isolated-clusters/defaults/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+isolated_clusters_ntp_image_name: ghcr.io/mwennrich/chrony
+isolated_clusters_ntp_image_tag:
+isolated_clusters_ntp_namespace: ntp-service
+
+isolated_clusters_dns_image_name: coredns/coredns
+isolated_clusters_dns_image_tag:
+isolated_clusters_dns_namespace: dns-service
+
+isolated_clusters_ingress_ingress_controller_namespace: ingress
```

Comment:
I think to shorten names, repeating "ingress" is not really necessary, it's still clear when it's `isolated_clusters_ingress_controller_namespace`.